test/upstream/mtk_structure_indexing.jl

```diff
@@ -156,9 +153,7 @@ end
 # Test integrator indexing.
 let 
     @test_broken false # NOTE: Multiple problems for `nint` (https://github.com/SciML/SciMLBase.jl/issues/662).
```

Comment:
technically issue is closed, but there is a follow-up regarding some (really niche) things that did not work and what do do about them.

Reply:
But why `@test_broken false`? Can we not just have a tracking issues with reference to where tests need updating?

Reply:
I plan to do that as well. However, I felt that it was useful to have some kind of identifier in the test code wherever their were broken tests so that we quickly can identify these places. For other broken tests we can just write `test_broken` before the actual test. However, here we test various integrators in a loop, and the test is only broken in some cases. Since there is a broken test here, I felt that adding `@test_broken false` was a good way to mark it.